packages/web-app-search/src/portals/SearchBar.vue

```diff
@@ -101,6 +101,7 @@ import Mark from 'mark.js'
 import debounce from 'lodash-es/debounce'
 import { eventBus } from 'web-pkg/src/services/eventBus'
 import { defineComponent } from '@vue/composition-api'
+import {mapGetters} from "vuex";
```

Comment:
Your autolinting looks broken 🤔

Reply:
Only this one file which does not lint correctly... what was the issue here again?

Reply:
Previously it was `script type="ts"` instead of `script lang="ts"`. But that's not the issue here. 🤔

Reply:
It seems that my IDE doesn't like the template structure of this file, hence it won't lint at all. `<li><oc-list><li>...</li></oc-list></li>`

![image](https://user-images.githubusercontent.com/50302941/197523854-c350b093-656a-4075-875b-ab89b05e2c6d.png)

Reply:
Ah, I have the same (in WebStorm).